app/build.gradle

```diff
@@ -48,19 +61,24 @@ dependencies {
     implementation "androidx.appcompat:appcompat:$appcompat_version"
     implementation "com.google.android.material:material:$material_version"
     implementation "androidx.constraintlayout:constraintlayout:$constraint_layout_version"
+    implementation 'androidx.test.ext:junit-ktx:1.1.3'
```

Comment:
might be nice to follow the same pattern of defining the version elsewhere

Reply:
thanks! I missed that when I updated the dependency. I accidentally used a support version, which is why I needed jetifier before.